src/drivers/distance_sensor/leddar_one/leddar_one.cpp

```diff
@@ -234,8 +259,11 @@ int leddar_one_main(int argc, char *argv[])
 
 		print_message(report);
 
+	} else if (!strcmp(verb, "info")) {
```

Comment:
So this just prints the help output. I'd rather remove it if the driver does not support any info output.

Reply:
Fair enough! Done.